File: java code/src/main/java/com/hfut/cqyzs/memorandum/bean/item/Item.java
package com.hfut.cqyzs.memorandum.bean.item;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.sql.Timestamp;

@Data
public class Item {
    @JsonProperty("itemId")
    int item_id;
    @JsonProperty("userId")
    String user_id;
    @JsonProperty("name")
    String name;
    @JsonProperty("isTrash")
    int is_trash;
    @JsonProperty("status")
    int status;
    @JsonProperty("createTime")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss",  timezone="GMT+8")
    String create_time;
}

File: android code/app/src/main/java/com/hfut/cqyzs/memorandum/local/entity/Item.kt
package com.hfut.cqyzs.memorandum.local.entity

import androidx.room.*
import com.google.gson.annotations.SerializedName

@Entity(
    tableName = "item",
    foreignKeys = [ForeignKey(
        entity = User::class,
        parentColumns = arrayOf("user_id"),
        childColumns = arrayOf("user_id"),
        onDelete = ForeignKey.CASCADE)],
    indices = [Index("user_id")]
)
data class Item (
    //用户名
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "item_id")
    @SerializedName("item_id")
    var itemId: Int,
    //用户名
    @ColumnInfo(name="user_id")
    @SerializedName("user_id")
    var userId:  String,
    //清单名称

    @ColumnInfo(name="name")
    var name:  String,

    @ColumnInfo(name="createTime")
    @SerializedName("create_time")
    var createTime: Long,
    //默认0（不在回收站） 1（在回收站）
    @ColumnInfo(name="isTrash")
    @SerializedName("is_trash")
    var isTrash:  Int,
    //同步状态
    @ColumnInfo(name="status")
    var status:  Int
)